bin/turbo_rspec

```diff
@@ -1,78 +1,71 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-ENV['RAILS_ENV'] ||= 'test'
+ENV["RAILS_ENV"] ||= "test"
 
-require './lib/turbo_tests'
-require 'optparse'
+require "./lib/turbo_tests"
+require "optparse"
 
 requires = []
 formatters = []
 verbose = false
 fail_fast = nil
 
-OptionParser.new do |opts|
-  opts.on("-r", "--require PATH", "Require a file.") do |filename|
-    requires << filename
-  end
+OptionParser
+  .new do |opts|
+    opts.on("-r", "--require PATH", "Require a file.") { |filename| requires << filename }
 
-  opts.on("-f", "--format FORMATTER", "Choose a formatter.") do |name|
-    formatters << {
-      name: name,
-      outputs: []
-    }
-  end
+    opts.on("-f", "--format FORMATTER", "Choose a formatter.") do |name|
+      formatters << { name: name, outputs: [] }
+    end
 
-  opts.on("-o", "--out FILE", "Write output to a file instead of $stdout") do |filename|
-    if formatters.empty?
-      formatters << {
-        name: "progress",
-        outputs: []
-      }
+    opts.on("-o", "--out FILE", "Write output to a file instead of $stdout") do |filename|
+      formatters << { name: "progress", outputs: [] } if formatters.empty?
+      formatters.last[:outputs] << filename
     end
-    formatters.last[:outputs] << filename
-  end
 
-  opts.on("-v", "--verbose", "More output") do
-    verbose = true
-  end
+    opts.on("-v", "--verbose", "More output") { verbose = true }
 
-  opts.on("--fail-fast=[N]") do |n|
-    n = Integer(n) rescue nil
-    fail_fast = (n.nil? || n < 1) ? 1 : n
+    opts.on("--fail-fast=[N]") do |n|
+      n =
+        begin
+          Integer(n)
+        rescue StandardError
+          nil
+        end
+      fail_fast = (n.nil? || n < 1) ? 1 : n
+    end
   end
-
-end.parse!(ARGV)
+  .parse!(ARGV)
 
 requires.each { |f| require(f) }
 
-if formatters.empty?
-  formatters << {
-    name: "progress",
-    outputs: []
-  }
-end
+formatters << { name: "progress", outputs: [] } if formatters.empty?
 
-formatters.each do |formatter|
-  if formatter[:outputs].empty?
-    formatter[:outputs] << '-'
-  end
-end
+formatters.each { |formatter| formatter[:outputs] << "-" if formatter[:outputs].empty? }
 
-# We do not want to include system specs by default, they are super
-# slow and require a selenium server to be running.
-default_spec_dirs = Dir.entries("#{Rails.root}/spec").reject do |entry|
-  !File.directory?("spec/#{entry}") || ["..", ".", "system"].include?(entry)
-end.map { |entry| "spec/#{entry}" }
-files = ARGV.empty? ? default_spec_dirs : ARGV
+if ARGV.empty?
```

Comment:
This and `TurboTests::Runner` below is the relevant part, I ran `stree` on this file too.